### ISSUE TITLE
Scaling png in `PolygonPatternSymbolizer`

### ISSUE THREAD

**Issue by zdila** [mapnik/mapnik-support#131]
Hello,

Is there a way to scale raster image in `PolygonPatternSymbolizer`? Attribute `transform="scale(0.5)" doesn't seem to work for rasters.

We don't want to save the scaled image directly because we want it to be crisp also for high-resolution rednering. And we can't use SVG because Mapnik renders the edges ugly if clipping is used.

**Comment by talaj** [comment]
Hi, this was reported recently in https://github.com/mapnik/mapnik-support/issues/129 and in https://github.com/mapnik/mapnik/issues/4097. It's really a shame this is not supported. I don't have time to work on it currently.
What's the problem with ugly edges and clipping?

**Comment by zdila** [comment]
Clipping is actually ignored. This is just a testcase:

![image](https://user-images.githubusercontent.com/636095/69324215-54511500-0c48-11ea-91a4-98cd95cc546e.png)

SVG file:

[orchard.svg.gz](https://github.com/mapnik/mapnik-support/files/3873848/orchard.svg.gz)

**Comment by lightmare** [comment]
Mapnik ignores SVG `clip-path`. That particular `orchard.svg` can be amended to not use clipping. Just crop the "apples" path by intersecting it with the clipPath rectangle.

**Comment by zdila** [comment]
Yes, I am cropping it now, but some patters were difficult to crop (in Inkscape).
